File: 1 - Begginer/C/1012 - Area.c
#include <stdio.h>
#include <math.h>
 
int main() {
    
    double pi = 3.14159;
    float A, B, C;
    
    scanf("%f %f %f", &A, &B, &C);
    
    float t_3, c, t_4, s, r;
    
    t_3 = A * C / 2;
    c = pi * pow(C, 2);
    t_4 = (A + B) * C / 2;
    s = pow(B, 2);
    r = A * B;
    
    printf("TRIANGULO: %.3f\nCIRCULO: %.3f\nTRAPEZIO: %.3f\nQUADRADO: %.3f\nRETANGULO: %.3f\n", t_3, c, t_4, s, r);
    
    return 0;
}

File: 1 - Begginer/Python/1012 - Area.py
pi = 3.14159

def read_line():
    return [float(x) for x in input().split()]

ABC = read_line()

t_3 = ABC[0] * ABC[2] / 2
c = pi * ABC[2] ** 2
t_4 = (ABC[0] + ABC[1]) * ABC[2] / 2
s = ABC[1] ** 2
r = ABC[0] * ABC[1]

print(f"TRIANGULO: {t_3:.3f}\nCIRCULO: {c:.3f}\nTRAPEZIO: {t_4:.3f}\nQUADRADO: {s:.3f}\nRETANGULO: {r:.3f}")